netbox/netbox/tables/columns.py

```diff
@@ -173,6 +173,7 @@ def __init__(self, *args, **kwargs):
             kwargs['attrs'] = {
                 'th': {
                     'class': 'w-1',
+                    'aria-label': 'Select all',
```

Comment:
Would it be better to put the label on the checkbox itself, rather than the `<th>` element?

Reply:
The error thrown was for a missing label in the table header itself rather than the checkbox within the table header